src/ui/Templates/TemplateCache.ts

```diff
@@ -10,19 +10,20 @@ import _ = require('underscore');
 export class TemplateCache {
   private static templates: { [templateName: string]: Template; } = {};
   private static templateNames: string[] = [];
+  private static pageTemplateNames: string[] = [];
```

Comment:
I'm not sure I like the naming "pageTemplate" ... Is it the best representation of what it does ?